packages/world/ts/config/v2/output.ts

```diff
@@ -4,13 +4,14 @@ import { DynamicResolution, ValueWithType } from "./dynamicResolution";
 import { Hex } from "viem";
 
 export type Module = {
-  /**
-   * The name of the module
-   * @deprecated
-   */
-  readonly name?: string;
   /** Should this module be installed as a root module? */
   readonly root: boolean;
+  /**
+   * Allow the module to call systems on your behalf.
+   * Useful when modules need to create namespaces or
+   * upgrade systems in namespaces you own.
+   */
+  readonly useDelegation: boolean;
```

Comment:
Do we throw an error/warning if both root and useDelegation are set to true?

Reply:
would be nice but we have no module config validation yet, nor super strong types, so didn't feel like adding it

what happens in this case is `root` is picked first, then `useDelegation`, then regular/default path

Reply:
these config options will ideally be abstracted away into helpers provided by the module like `defineERC20Module`

Reply:
could make sense to use the same `installStrategy: "root" | "delegation" | "default"` approach in config but will tackle that later when we clean up module config (needs a rethink for args anyway: https://github.com/latticexyz/mud/issues/2957)